microprofile/WFLY-14261_upgrade-microprofile-health-3.0.adoc

```diff
@@ -0,0 +1,79 @@
+= Upgrade MicroProfile Health to 3.0
+:author:            Martin Stefanko
+:email:             mstefank@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          microprofile,health,observability
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+
+MicroProfile 4.0 provides a major upgrade for https://github.com/eclipse/microprofile-health/releases/tag/3.0[MicroProfile Health 3.0] with changes described in its https://download.eclipse.org/microprofile/microprofile-health-3.0/microprofile-health-spec-3.0.html#release_notes_3_0[Release Notes].
+
+This component upgrade also covers the upgrade of smallrye-health 3.0.0 that implements MicroProfile Health 3.0.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/WFLY-14261[WFLY-14261]
+
+=== Related Issues
+
+* https://issues.jboss.org/browse/EAP7-1593[EAP7-1593]
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+* TODO
+
+=== Testing By
+
+[X] Engineering
+[ ] QE
+
+== Requirements
+
+* Upgrade MicroProfile Health to 3.0
+* Upgrade SmallRye Health to 3.0.0
+* Pass all MicroProfile Health tests in WildFly integration test suite
+* Pass the MicroProfile Health 3.0 TCK without failures
+* Adjust the tests to work as needed
+* Adjust QuickStarts to address backward compatibility issues (tracked in https://issues.redhat.com/browse/WFLY-14398[WFLY-14398])
```

Comment:
If quickstart change is part of this RFE, we would need change of quickstart test as well?

Reply:
only change required in QS was this - https://github.com/wildfly/quickstart/pull/470 so use standalone-microprofile.xml for arquillian tests. Do you think that precheck or QS testsuite is necessary in this regard?